libs/langchain/langchain/chat_models/fake.py

```diff
@@ -14,6 +16,7 @@ class FakeListChatModel(SimpleChatModel):
     """Fake ChatModel for testing purposes."""
 
     responses: List
+    sleep: Optional[float] = None
```

Comment:
Could you document that sleep should generally not be used?

Reply:
Well fake llm should generally not be used?

Reply:
generally i think we want to discourage usage of sleep in unit tests as there's almost always a better way of writing the unit test (this code being an exception if we're trying to yield control between tasks/threads)

Reply:
Just tried sleep 0 and that doesnt work (ie. the test doesnt fail when it should)